tasks/ldap.yml

```diff
@@ -2,26 +2,26 @@
 # vim: foldmarker=[[[,]]]:foldmethod=marker
 
 - name: Check default ldap config exists
-  command: php --file "{{ owncloud__home }}/occ" ldap:show-config ""
+  command: '{{ owncloud__occ_bin_file_path | quote }} ldap:show-config'
   changed_when: False
   when: (not ansible_check_mode)
   register: owncloud__register_ldap_default_config_exits
   become_user: '{{ owncloud__user }}'
```

Comment:
Mmm, ok, the problem is:

```
# Default, access with root without sudo
ansible_ssh_user: root
ansible_become: false
```

Without this vars it works. Maybe i had this vars for older ansible versions.

Anyway, do you prefer use OCC custom command or revert this changes?